ownership/ownership.go

```diff
@@ -12,112 +12,114 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package keysync
+package ownership
 
 import (
-	"bufio"
 	"fmt"
-	"os"
+	"os/user"
 	"strconv"
-	"strings"
 
 	"github.com/sirupsen/logrus"
 )
 
+type lookup interface {
```

Comment:
this feels like it should be exported, especially that it's part of the `NewOwnership` function signature, and `Lookup` is an exported field of the `Ownership` struct.

Reply:
Yeah, seems fine.

Reply:
Strictly speaking I don't think it's needed (since you could provide an alternative implementation without naming the interface) but there's no particular reason to not make it exported.